triton-server/config.ts

```diff
@@ -1,25 +1,24 @@
-/*
- *
- * This is an example configuration. The application requires a similar file,
- * named 'config.ts' in the same folder as this one, in order to work.
- */
-
 import path from 'path'
 import 'dotenv/config'
 
 export const {
+	API_URL = 'http://localhost:3001',
 	LOGGER_LEVEL = 'info',
 	CLIENT_ORIGIN = 'http://localhost:3000',
 	CLIENT_PORTAL_ORIGIN = 'http://localhost:1234',
-	CLIENT_PORTAL_USERNAME = 'wxGFi8MjHlakuaQ-ls99Rw..',
-	CLIENT_PORTAL_PASSWORD = 'eW4yvfLHi5nlRNn8EJlsPg..',
+	CLIENT_PORTAL_USERNAME = '',
+	CLIENT_PORTAL_PASSWORD = '',
 	LIMS_API_URL = 'http://127.0.0.1:8000/api',
-	LIMS_USERNAME = 'potato',
-	LIMS_PASSWORD = 'potato',
+	LIMS_USERNAME = '',
+	LIMS_PASSWORD = '',
+	SFTP_SERVER = '0.0.0.0',
+	SFTP_PORT = '21',
+	SFTP_DATA_PREFIX = '/data/glsftp/',
```

Comment:
don't think this is used anymore this is legacy code that connected to clarity ...